### DISCUSSION TITLE
Notebook kernel crashes when reading large parquet files.

### DISCUSSION THREAD

**Discussion by GabrielKuka** [duckdb/duckdb#7385] [General]
I'm trying to read large parquet files and load them in a table in Duckdb. The parquet files contain over 100 million rows.

Code: 

_import duckdb
con = duckdb.connet(":memory:")
data_path="../october_10days/*.parquet"
data = con.from_parquet(data_path)
PRAGMA memory_limit='8gb'
CREATE TABLE nested_data AS SELECT * FROM data_

When creating the table the kernel crashes.

I'm running: Ubuntu 23.04 x64
Duckdb 0.7.1

**Comment by Alex-Monahan** [comment]
Hello! If you are working with more data than you have RAM, we recommend using a disk-backed DuckDB! As an alternative, you could set the temp_directory, but that may be slower than just writing straight to disk.

**Reply by GabrielKuka** [reply]
I removed the memory limit and created the db on disk. I still the get same issue.

**Comment by l1t1** [comment]
does this happen when only one parquet file in that directory?

**Reply by GabrielKuka** [reply]
This happens when using all the parquet files in the directory and also if I use just one parquet file.

**Reply by carlopi** [reply]
Could you try to reproduce using latest nightly (here the artifacts: https://github.com/duckdb/duckdb/actions/runs/4975805806)?
There have been plenty of bugfixes since 0.7.1.

**Reply by GabrielKuka** [reply]
I tried now with version 0.8.1 and the notebook is not crashing anymore. :) Thanks.
